packages/gestalt/src/IconButton.tsx

```diff
@@ -55,7 +62,7 @@ type Props = {
   /**
    * Primary color to apply to the [Icon](/web/icon). See [icon color](https://gestalt.pinterest.systems/web/iconbutton#Icon-color) variant to learn more.
    */
-  iconColor?: 'gray' | 'darkGray' | 'red' | 'white' | 'brandPrimary';
+  iconColor?: 'dark' | 'gray' | 'darkGray' | 'red' | 'white' | 'brandPrimary';
```

Comment:
![Screenshot by Dropbox Capture](https://github.com/pinterest/gestalt/assets/10593890/2893fdce-8e1f-4c65-bf6c-aa688116012b)


darkGray is 1111111 yu might be duplicating it